Apps/Src/SendTritium.c

```diff
@@ -668,6 +665,8 @@ void Task_SendTritium(void *p_arg){
         #endif
         state.stateDecider();    // decide what the next state is
 
+        velocitySetpoint = (velocitySetpoint>0)?MAX_VELOCITY:-MAX_VELOCITY;
```

Comment:
I'll add a comment to explain it, but it basically disables regen. The velocity setpoint is always set to the maximum (unreachable) value in whichever direction we are traveling, which ensures that we don't ever enter velocity controlled mode (and stay in current controlled mode) while still being allowed to test regen/cruise mode transition logic since the transition itself isn't explicitly disabled.

Reply:
Ah, thanks for explaining it!